cmd/fluxsvc/main.go

```diff
@@ -56,6 +59,24 @@ func main() {
 		os.Exit(0)
 	}
 
+	// If the events-url flag is present, ignore instance specific Slack settings.
+	var defaultEventsConfig *instance.Config
+	if eventsURL != nil {
```

Comment:
I don't think this will ever be nil (since pflags will give it the default, by now). But `*eventsURL` may be an empty string.